Fred2/test/TestTranscript.py

```diff
@@ -25,13 +25,13 @@ def setUp(self):
         self.w_v = Transcript(self.gcg_ts, 'GLUC_HUMAN', "NM_002054.4", {343: self.gcg_v1})
 
     def test_consistency(self):
+        #TODO: FYI if run with all other tests, Transcript.newid() that counts num of transcripts and defines ID does not match (is 15) and test does not run through.
```

Comment:
Line 33 in Transcript.py is indeed evil. When running the regular tests or only this test, the Transcript.newid() index corresponds with the expected values and starts at zero when the Transcripts for this test are initiated. Running the external tests as well, leads to a newid() that starts with 15 (I guess 15 transcripts have been initiated before with their own ids) not matching the expected representation and leading to a AssertionError.